### DISCUSSION TITLE
JSON license in eslint - 8.22.0

### DISCUSSION THREAD

**Discussion by rahulmohang** [eslint/eslint#17144] [General] [answered]
Hi, 

The license of the following files is the JSON license:

https://github.com/eslint/eslint/blob/main/tests/bench/large.js
https://github.com/eslint/eslint/blob/main/tests/performance/jshint.js

The JSON license, that can be found at https://www.json.org/license.html, is considered ambiguous as per open source foundations like the Eclipse Foundation and the Apache Foundation. Please see an article related to this at https://www.synopsys.com/blogs/software-security/json-license-limitations/. 

Is it possible to relicense the files?

Regards,
Rahul

**Comment by Rec0iL99** [comment]
#17143 is a similar issue.

**Reply by rahulmohang** [reply]
Thank you.

**Comment by nzakas** [comment, marked as answer]
I answered on the issue as well, but basically, these files shouldn't matter. They are part of the test suite and not distributed with ESLint when published.

**Reply by rahulmohang** [reply]
Thank you.

**Reply by stbischof** [reply]
https://github.com/eslint/eslint/pull/17165
